src/marketplace/MakeAComponentSaleView.js

```diff
@@ -0,0 +1,93 @@
+//@flow strict
+import * as React from 'react';
+import Cookies from 'universal-cookie';
+
+type Props = {||}; 
+type State = {|
+    userId: string,
+    salePrice: number,
+    itemID: string,
+    itemAmount: number,
+|};
+let itemsToSell = [{value: '', label: ''}];
```

Comment:
I've talked about why this shouldn't be a thing many times so far.
Also what does value and label mean?
Also what is the type of items to sell?
Also why are you adding empty things to the list?